lib/src/sprite.dart

```diff
@@ -1,5 +1,7 @@
 import 'dart:ui';
 
+import '../image_composition.dart';
+
```

Comment:
extra empty line :p 

the import blocks are flutter/ext libraries and relative imports (that are separated by empty lines)